src/Core/Framework/DependencyInjection/seo.xml

```diff
@@ -154,11 +157,5 @@
         </service>
 
         <service id="Shopware\Core\Content\Seo\SeoUrlTwigFactory"/>
-
-        <service id="shopware.seo_url.twig" class="Twig\Environment">
```

Comment:
this is an BC removing services

Reply:
I know. Did you read my changelog?

Reply:
And? it doesn't matter. you can't break anything in trunk.

Reply:
Oh, sorry. I thought it is your task to merge it so you don't break stuff. Ok ok ok. So which branch will I choose instead?